app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeViewModel.kt

```diff
@@ -89,13 +89,18 @@ class ComposeViewModel @Inject constructor(
     val markMediaAsSensitive: MutableStateFlow<Boolean> =
         MutableStateFlow(accountManager.activeAccount?.defaultMediaSensitivity ?: false)
 
-    val statusVisibility: MutableStateFlow<Status.Visibility> = MutableStateFlow(Status.Visibility.UNKNOWN)
+    val statusVisibility: MutableStateFlow<Status.Visibility> =
```

Comment:
Many of the changes here appear to be formatting changes, not functionality changes. That makes it difficult to review the code that's actually making changes. @evant would you mind reverting the formatting changes on existing code, and leaving just the functional changes? Thanks.